package.json

```
@@ -12,15 +12,12 @@
     "jupyterlab-extension"
   ],
   "scripts": {
-    "build": "jlpm run build:lib",
-    "build:labextension": "cd jupyterlab_git && rimraf labextension && mkdirp labextension && cd labextension && npm pack ../..",
-    "build:lib": "tsc",
-    "build:all": "jlpm run build:labextension",
-    "clean": "jlpm run clean:lib",
-    "clean:lib": "rimraf lib && rimraf tsconfig.tsbuildinfo",
+    "build": "jlpm install && tsc",
+    "build:labextension": "jlpm build && jlpm clean:labextension && mkdirp jupyterlab_git/labextension && cd jupyterlab_git/labextension && npm pack ../..",
+    "clean": "rimraf lib && rimraf tsconfig.tsbuildinfo",
```

I discovered recently that this can be simplified
```suggestion
    "clean": "rimraf lib tsconfig.tsbuildinfo",
```

---

I was wondering about that myself when I was working on this, but never got around to trying it out. Thanks!

---

Done